contracts/DappRegistry.sol

```diff
@@ -0,0 +1,102 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+
+import "@ensdomains/ens-contracts/contracts/registry/ENS.sol";
+import "./FDSRegistrar.sol";
+
+contract DappRegistry {
+
+	struct Record {
+		// ENS app name
+		bytes32 node;
```

Comment:
why is this necessary? we just store records in the `records` mapping where you need to pass the `node` as key.